test/markup_test.rb

```diff
@@ -93,7 +91,7 @@ def test_each_render_has_a_name
     assert_equal "wikicloth", GitHub::Markup.renderer('README.wiki', '<h1>Title</h1>').name
     assert_equal "asciidoctor", GitHub::Markup.renderer('README.adoc', '== Title').name
     assert_equal "restructuredtext", GitHub::Markup.renderer('README.rst', 'Title').name
-    assert_equal "pod", GitHub::Markup.renderer('README.pod', '=begin').name
+    assert_equal "pod", GitHub::Markup.renderer('README.pod', '=head1').name
```

Comment:
Shall we also add something like this?
```
assert_equal "pod6", GitHub::Markup.renderer('README.pod6', '=begin pod').name
```

Reply:
We could, right...

Reply:
I've added this, though it breaks without the corresponding change in Linguist being used (github/linguist#4149). Will resolve that next.